R/utilities-aesthetics.R

```diff
@@ -86,14 +86,34 @@ asPlotShape <- function(shapes) {
 }
 
 #' @title ColorMaps
-#' @description List with some color maps for `Theme` object
+#'
+#' @description
+#'
+#' List with some color maps for `Theme` object.
+#'
+#' The `ospDefault` color map is based on colors in `default_igv` qualitative
+#' color palette from `{ggsci}` package.
+#'
 #' @export
 ColorMaps <- list(
   default = c("#0078D7", "#D83B01", "#107C10", "#A80000", "#002050", "#B4009E"),
   grays = paste("gray", seq(0, 100, 10), sep = ""),
   prism = c("#FF0000", "#FF7F00", "#FFFF00", "#00FF00", "#0000FF", "#4B0082", "#8F00FF"),
   blot = c("blue", "magenta", "cyan", "green", "yellow", "red"),
-  temperature = c("#262A76", "#234990", "#2F8AC3", "#26B0D2", "#FFC1CB", "#EB559", "#AE3535", "8E1F20")
+  temperature = c("#262A76", "#234990", "#2F8AC3", "#26B0D2", "#FFC1CB", "#EB559", "#AE3535", "8E1F20"),
+  ospDefault = c(
+    "#5050FFFF", "#CE3D32FF", "#749B58FF", "#F0E685FF",
+    "#466983FF", "#BA6338FF", "#5DB1DDFF", "#802268FF", "#6BD76BFF",
+    "#D595A7FF", "#924822FF", "#837B8DFF", "#C75127FF", "#D58F5CFF",
+    "#7A65A5FF", "#E4AF69FF", "#3B1B53FF", "#CDDEB7FF", "#612A79FF",
+    "#AE1F63FF", "#E7C76FFF", "#5A655EFF", "#CC9900FF", "#99CC00FF",
+    "#A9A9A9FF", "#CC9900FF", "#99CC00FF", "#33CC00FF", "#00CC33FF",
+    "#00CC99FF", "#0099CCFF", "#0A47FFFF", "#4775FFFF", "#FFC20AFF",
+    "#FFD147FF", "#990033FF", "#991A00FF", "#996600FF", "#809900FF",
+    "#339900FF", "#00991AFF", "#009966FF", "#008099FF", "#003399FF",
+    "#1A0099FF", "#660099FF", "#990080FF", "#D60047FF", "#FF1463FF",
+    "#00D68FFF", "#14FFB1FF"
+  )

```

Comment:
just out of curiosity: are those magic numbers created randomly or is there any kind of "algorithm" behind?

Reply:
You can see the algorithm here:
https://rdrr.io/cran/ggsci/src/R/discrete-igv.R

Reply:
Thanks. Following that link I found the page here with the nice overview of those colour palettes:
https://cran.r-project.org/web/packages/ggsci/vignettes/ggsci.html

Which gives some inspiration for our future extensions :)

Reply:
If you are looking for a more general overview of color palettes in R, you can see it here:
https://emilhvitfeldt.github.io/paletteer/

It combines palettes from dozens of different packages.

Reply:
Nice. We should suggest https://github.com/aljrico/gameofthrones as our default palette to @PavelBal 😉